SimG4CMS/HGCalTestBeam/test/waferGeom.C

```diff
@@ -0,0 +1,75 @@
+#include <algorithm>
```

Comment:
@bsunanda - I'd suggest to add a macros directory and put it there. For example: http://cmslxr.fnal.gov/source/Fireworks/Geometry/macros/